src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/blocks/needbraces/Example2.txt

```diff
@@ -26,6 +26,6 @@ while (counter < 10)                            // OK
     ++count;
 while (value.incrementValue() < 5); // OK
 switch (num) {
-  case 1: counter++; break;         // OK
+  case 1: counter++; break;        // OK, 'break' is not counted to allow compact view
```

Comment:
```suggestion
  case 1: counter++; break;        // OK, 'LITERAL_CASE' is not in configured tokens
```

Either this or just simple `// ok` comment, counting breaks has nothing to do with the lack of violation here.